mungegithub/mungers/close-stale-obj.go

```diff
@@ -30,83 +30,55 @@ import (
+	stalePeriod    = 90 * day // Close the PR/Issue if no human interaction for `stalePeriod`
+	startWarning   = 60 * day
+	remindWarning  = 30 * day
+	closingComment = `This PR/Issue hasn't been active in %s. Closing this PR/Issue. Please reopen if you would like to work towards merging this change, if/when the PR/Issue is ready for the next round of review.
 
 %s
 You can add 'keep-open' label to prevent this from happening again, or add a comment to keep it open another 90 days`
-	warningComment = `This PR hasn't been active in %s. It will be closed in %s (%s).
+	warningComment = `This PR/Issue hasn't been active in %s. It will be closed in %s (%s).
 
 %s
 You can add 'keep-open' label to prevent this from happening, or add a comment to keep it open another 90 days`
 )
 
 var (
-	closingCommentRE = regexp.MustCompile(`This PR hasn't been active in \d+ days?\..*label to prevent this from happening again`)
-	warningCommentRE = regexp.MustCompile(`This PR hasn't been active in \d+ days?\..*be closed in \d+ days?`)
+	closingCommentRE = regexp.MustCompile(`This PR/Issue hasn't been active in \d+ days?\..*label to prevent this from happening again`)
+	warningCommentRE = regexp.MustCompile(`This PR/Issue hasn't been active in \d+ days?\..*be closed in \d+ days?`)
 )
 
-// CloseStalePR will ask the Bot to close any PullRequest that didn't
-// have any human interactions in `stalePullRequest` duration.
+// CloseStaleObj will ask the Bot to close any PR/Issue that didn't
+// have any human interactions in `stalePeriod` duration.
 //
 // This is done by checking both review and issue comments, and by
-// ignoring comments done with a bot name. We also consider re-open on the PR.
-type CloseStalePR struct{}
+// ignoring comments done with a bot name. We also consider re-open on the PR/Issue.
+type CloseStaleObj struct{}
 
 func init() {
-	s := CloseStalePR{}
+	s := CloseStaleObj{}
 	RegisterMungerOrDie(s)
 	RegisterStaleComments(s)
 }
 
 // Name is the name usable in --pr-mungers
-func (CloseStalePR) Name() string { return "close-stale-pr" }
+func (CloseStaleObj) Name() string { return "close-stale-obj" }
 
 // RequiredFeatures is a slice of 'features' that must be provided
-func (CloseStalePR) RequiredFeatures() []string { return []string{} }
+func (CloseStaleObj) RequiredFeatures() []string { return []string{} }
 
 // Initialize will initialize the munger
-func (CloseStalePR) Initialize(config *github.Config, features *features.Features) error {
+func (CloseStaleObj) Initialize(config *github.Config, features *features.Features) error {
 	return nil
 }
 
 // EachLoop is called at the start of every munge loop
-func (CloseStalePR) EachLoop() error { return nil }
+func (CloseStaleObj) EachLoop() error { return nil }
 
 // AddFlags will add any request flags to the cobra `cmd`
-func (CloseStalePR) AddFlags(cmd *cobra.Command, config *github.Config) {}
-
-func findLastHumanPullRequestUpdate(obj *github.MungeObject) (*time.Time, bool) {
```

Comment:
I mean findLastHumanPullRequestUpdate is still needed to not consider PRs stale when their activity only happens in Pull Request Reviews.

And AddFlags is necessary to match the Munger interface-- apparently the tests don't catch that!